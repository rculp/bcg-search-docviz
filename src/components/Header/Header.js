import React, { Component } from 'react';

import './Header.css';

class Header extends Component {
  render() {
    return (
      <header>Minerva</header>
    );
  }
}

export default Header;