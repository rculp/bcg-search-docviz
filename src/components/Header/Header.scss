@import 'src/global-styles/colors';
@import 'src/global-styles/responsive';

.page-header {
  width: 100%;
  height: 56px;
  font-weight: bold;
  text-align: center;

  @media (max-width: map-get($grid-breakpoints, sm)) {
    height: 48px;
  }

  .logo {
    font-family: 'BCG-Henderson-Sans';
    font-size: 2.8rem;
    width: 56px;
    height: 56px;
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: $bcg-green--dark;
    color: $white;
    text-align: center;

    @media (max-width: map-get($grid-breakpoints, sm)) {
      width: 48px;
      height: 48px;
      font-size: 2.2rem;
    }
  }
}
