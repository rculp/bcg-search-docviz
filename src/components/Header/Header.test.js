import { shallow } from 'enzyme';

import React from 'react';

import Header from './Header';

describe(Header, () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Header />
    );
    expect(component).toMatchSnapshot();
  });
});
