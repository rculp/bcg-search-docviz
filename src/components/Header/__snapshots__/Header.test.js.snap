// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`() =>
_react2.default.createElement('header', { __source: { fileName: _jsxFileName, lineNumber: 6 }, __self: undefined }, 'Minerva') renders and matches our snapshot 1`] = `
<header>
  Minerva
</header>
`;
